includes/Helpers/ManageWikiExtensions.php

```diff
@@ -90,16 +98,18 @@ public function add( $extensions ) {
 	/**
 	 * Removes an extension from the 'enabled' list
 	 * @param string|string[] $extensions Either an array or string of extensions to disable
+	 * @param bool $forceRemove Force removing extension incase it is removed from config
 	 */
-	public function remove( $extensions ) {
+	public function remove( $extensions, $forceRemove ) {
```

Comment:
pass false to forceRemove in overwiteAll below or this will error

Reply:
I supplied a default value of false to it.